ColorCodeReferenceManual.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+
+const char* MajorColorNames[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* MinorColorNames[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
+int numberOfMajorColors = sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
+int numberOfMinorColors = sizeof(MinorColorNames) / sizeof(MinorColorNames[0]);
+
+void PrintColorCodingReference() {
+    int pairNumber = 1;
```

Comment:
how to test PrintColorCodingReference ?
Use camel case for naming functions.
Kindy go through the requirement again , Which new feature needs to be implemented ? printing or preparing manual for printing?